ddtrace/contrib/pylons/middleware.py

```diff
@@ -107,11 +107,11 @@ def __call__(self, environ, start_response):
                         if hasattr(request, "json"):
                             req_body = request.json
                         else:
-                            req_body = json.loads(request.body.decode("UTF-8"))
+                            req_body = json.loads(request.body.decode("UTF-8", "ignore"))
```

Comment:
Question: `xmltodict.parse` uses uft8 by default but it can [handle different encodings](https://omz-software.com/pythonista/docs/ios/xmltodict.html). Instead of ignoring errors can we use [request.charset](https://github.com/Pylons/webob/blob/main/src/webob/request.py#L131) to get the correct encoding?